ros_speech_recognition/sample/sample_ros_speech_recognition.launch

```diff
@@ -0,0 +1,23 @@
+<launch>
+
+  <arg name="google_cloud_credentials_json" default="''" doc="Credential JSON is only used when the engine is GoogleCloud." />
+  <arg name="engine" default="GoogleCloud" doc="Speech to text engine. TTS engine, Google, GoogleCloud, Sphinx, Wit, Bing Houndify, IBM" />
```

Comment:
In my understanding, `GoogleCloud` needs credentials. (It costs money)
I think the default engine should be `Google` so that we can try this ROS package for free.

If you think `GoogleCloud` should be the default for `ros_speech_recognition`, I think it's ok to keep this change.